.Rbuildignore

```diff
@@ -43,3 +43,5 @@ _files$
 ^compile_commands\.json$
 ^\.cache$
 ^tests/testthat/helper-rethrow\.R$
+^cleanup$
+^cleanup\.win$
```

Comment:
Is it correct that these are here?

Reply:
I think so, the tarball has the `.xz` file and doesn't need this procedure.

Reply:
`cleanup` is run as part of `R CMD build .` in the right location at the right time. Seems legit, but 🤷 .